server/index.js

```diff
@@ -19,7 +19,7 @@ app.get('/api/rand', (req, res) => {
 });
 
 app.get('/api/featured', (req, res) => {
-  let index = req.body.index;
+  let index = req.query.index;
   db.getFeaturedFilmById(index, (err, result) => {
     res.status(200).send(result);
```

Comment:
If we were dealing with a lot more data, it may be faster (from the client's perspective) to filter the unnecessary elements server-side.

Reply:
good idea, luckily the getFeatureFilmById returns an array with only a single entry in it so I don't think it will make that much of a difference